### PR TITLE
init base project

### PR DESCRIPTION
**Pull request by HaLanHuynh**
Init base project.

Link: https://edu-redmine.sun-asterisk.vn/issues/10052

### CODE REVIEW

**Comment by HaLanHuynh** [comment]
ready

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.framgia.weather04"
```
Comment (tuanlvt-1117) [inline app/build.gradle]: cty đổi tên rồi e
```suggestion
        applicationId "com.sun.weather04"
```

File: app/src/main/java/com/framgia/weather04/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import android.view.MenuItem;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+    }
+
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/weather04/MainActivity.java]: remove blank

**Comment by HaLanHuynh** [comment]
ready

**Pull request merged**